spec_02.md

```diff
@@ -0,0 +1,95 @@
+- Spec: 2
+- Title: Methods on figure objects
+- Version: 0.1
+- Last-Modified: 2019-07-19
+- Authors: [James Bednar](),
+           [David Hoese](),
+           [Jon Mease](jon.mease@gmail.com),
+           [Julia Signell](jsignell@gmail.com),
+- Status: Active
+- Type: Standards
+- Content-Type: text/markdown
+- Created: 2019-07-19
+
+## Abstract
+This spec describes a proposed shared Python API for accessing objects from various plotting libraries.
+
+In this notebook we will use **figure** to refer to any object that has a visible representation but that can also be saved, exported, etc. This includes generic visualizations or animations that may or may not be considered a "plot" with axes, ticks, and so on.```
+
+When detailing a method, this spec uses FIGURE to refer to any specific figure object and LIBRARY to refer to any specific visualization library.
+
+## If a method doesn't make sense
+If a given operation doesn't make sense for that library, then it can satisfy the spec by simply having that method return a message to that effect ("Unsupported: LIBRARY does not provide JSON output").
+
+## Jupyter methods
+Every library that supports rendering in Jupyter should support at least one of the various IPython rich display methods, i.e. `_repr_html_`, `_repr_png_`, `_ipython_display_`.
```

Comment:
I believe ``_repr_mimebundle_`` along with a separately-installed frontend extension is the recommended way to do this according to the Jupyter team.

Reply:
Do you propose just adding `_repr_mimebundle_` to the list, or enforcing that it should be implemented?

Reply:
I think adding it to the list is fine, but folks from the Jupyter team may have stronger opinions.